crates/bitwarden/src/auth/mod.rs

```diff
@@ -1,2 +1,4 @@
+pub mod api;
```

Comment:
This should probably not be public, however we use api response in captcha response. Might be worth figuring that out.

Reply:
Would using `pub(super) mod api;` help here? That way it'll only be accesible to the auth module.